Card/src/main/java/com/braintreepayments/api/Card.java

```diff
@@ -70,6 +86,16 @@ public void setMerchantAccountId(String id) {
         merchantAccountId = TextUtils.isEmpty(id) ? null : id;
     }
 
+    /**
+     * @param shouldValidate Flag to denote when the associated {@link PaymentMethodNonce}
+     *                       will be validated. When set to {@code true}, the {@link PaymentMethodNonce}
+     *                       will be validated immediately. When {@code false}, the {@link PaymentMethodNonce}
+     *                       will be validated when used by a server side library for a Braintree gateway action. Defaults to false.
+     */
```

Comment:
I think can align our docs comments for `shouldValidate` across SDKs better (see [iOS](https://github.com/braintree/braintree_ios/blob/354475775cc745c9e571cd154437e9812cac5266/Sources/BraintreeCard/Public/BraintreeCard/BTCard.h#L110-L115)).

In general I think the `validate` concept is confusing and I'm not sure which docstring is more accurate - but wanted to raise the concern.

Reply:
Good callout 🙏 . Added an additional note in the docs similar to iOS.